content/upgrade-pop.md

```diff
@@ -87,19 +89,21 @@ sudo apt update
 sudo apt full-upgrade
 ```
 
-Once the updates are applied, a notification should appear at the top of your screen saying that an upgrade is available. Click on this notification, or go to Settings ->  OS Upgrade & Recovery. The System76 upgrade package will display a message that Pop!\_OS 21.10 is available with a `Download` button.
+Once the updates are applied, a notification should appear at the top of your screen saying that an upgrade is available. Click on this notification, or go to Settings ->  OS Upgrade & Recovery. The System76 upgrade package will display a message that Pop!\_OS 22.04 is available with a `Download` button.
 
 If you are planning on staying on an LTS release for the time being, this is also the page where you can dismiss upgrade notifications.
 
-![Settings OS Upgrade](/images/upgrade-pop/setting-os-upgrade.png)
+The OS Upgrade and Recovery page of settings now allows for automatic update scheduling and updating the Recovery partition as well.
```

Comment:
I'm wondering how relevant this line is to the upgrade process this article is covering. Mentioning Recovery might not be entirely necessary since that's performed automatically as part of a release upgrade.

Reply:
If the upgrade is failing, updating Recovery and then refreshing will fix the issue.

Reply:
That's fair.

Reply:
That's fair, if a little out of context here. How about automatic update scheduling, is that relevant to performing a release upgrade?

If we do keep the sentence, could it be combined with the previous sentence into a paragraph? Having "also" in one sentence and then "as well" in the next, on separate lines, makes the section sound long-winded, imo. If this one's following up on the previous one, they should at least be grouped together.